docs/translations/README.md

```diff
@@ -0,0 +1,22 @@
+# Translations
+
+This folder contains documentation around translation handling of WooCommerce Blocks.
+
+| Document                                                          | Description                                                    |
```

Comment:
The repeated use of `This doc...` in the description column seems redundant, maybe we could remove it? (I would also argue that the description column could be done away with entirely, but it seems present in all the other documents so it should stay for consistency)

Reply:
I agree that this is redundant and changed the table into a list of files for navigation purposes.